### PR TITLE
create basic express app that runs the generator

### PR DESCRIPTION
**Pull request by championswimmer**
check this PR out,

locally run
```
npm install
npm run server.generator
```

Visit localhost:3000 -> you'll see the form for generator
Visit localhost:3000/generate -> you'll see OTS16 data schedule page

Signed-off-by: Arnav Gupta <championswimmer@gmail.com>

### CODE REVIEW

File: src/generator/backend/generator.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  else return '0' + num.toString()
+}
+
+function foldByTrack(sessions, speakers) {
+  let trackData = new Map()
+  let speakersMap = new Map(speakers.map(s => [s.id, s]));
+
+  sessions.forEach(session => {
+    if (!session.start_time)
+      return
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l84)

File: src/generator/backend/generator.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  if (str === undefined) {
+    return ""
+  }
+  return str.replace(/[^\w]/g, '-').replace(/-+/g, '-').toLowerCase()
+}
+
+function speakerNameWithOrg(speaker) {
+  return speaker.organisation ?
+    `${speaker.name} (${speaker.organisation})` :
+    speaker.name
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l42)

File: src/generator/backend/generator.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
+'use strict'
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l1)

File: src/generator/backend/generator.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+var exports = module.exports = {};
+
+const fs = require('fs')
+const moment = require('moment')
+const handlebars = require('handlebars')
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l7)

File: src/generator/backend/generator.js [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+const tpl = handlebars.compile(fs.readFileSync(__dirname + '/schedule.tpl').toString('utf-8'))
+const sessionsData = require('../../../mockjson/sessions.json')
+const speakersData = require('../../../mockjson/speakers.json')
+const servicesData = require('../../../mockjson/event.json')
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l12)

File: src/generator/backend/generator.js [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+    if (a[key] < b[key]) return -1
+    else return 0
+  }
+}
+
+function zeroFill(num) {
+  if (num === undefined) {
+    return ""
+  }
+  if (num >= 10) return num.toString()
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l74)

File: src/generator/backend/generator.js [hunk 7]
```diff
@@ -0,0 +1,234 @@
+'use strict'
+
+var exports = module.exports = {};
+
+const fs = require('fs')
+const moment = require('moment')
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l6)

File: src/generator/backend/generator.js [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+handlebars.registerHelper('linkify', function (options) {
+  var content = options.fn(this);
+  return new handlebars.SafeString(content.linkify());
+});
+
+function slugify(str) {
+  if (str === undefined) {
+    return ""
+  }
+  return str.replace(/[^\w]/g, '-').replace(/-+/g, '-').toLowerCase()
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l36)

File: src/generator/backend/generator.js [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+
+    track.sessions.push({
+      start: moment(session.start_time).utcOffset(2).format('HH:mm'),
+      title: session.title,
+      type: session.type,
+      location: session.location,
+      speakers: session.speakers.map(speakerNameWithOrg).join(', '),
+      speakers_list: session.speakers.map(speaker => speakersMap.get(speaker.id)),
+      description: session.description,
+      session_id: session.session_id,
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Identifier 'session_id' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l113)

File: src/generator/backend/generator.js [hunk 10]
```diff
@@ -0,0 +1,234 @@
+'use strict'
+
+var exports = module.exports = {};
+
+const fs = require('fs')
```
Comment (juslee) [inline src/generator/backend/generator.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2699255326/issues/source?bid=3233857&fileBranchId=3341539#l5)

**Comment by championswimmer** [comment]
@aayusharora  @mariobehling @juslee 
Do not merge yet, till the code quality issues are resolved

**Pull request merged**